x-pack/plugins/threat_intelligence/cypress/e2e/indicators.cy.ts

```diff
@@ -45,29 +48,68 @@ const THREAT_INTELLIGENCE = '/app/security/threat_intelligence/indicators';
 const URL_WITH_CONTRADICTORY_FILTERS =
   '/app/security/threat_intelligence/indicators?indicators=(filterQuery:(language:kuery,query:%27%27),filters:!((%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%27%27,key:threat.indicator.type,negate:!f,params:(query:file),type:phrase),query:(match_phrase:(threat.indicator.type:file))),(%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%27%27,key:threat.indicator.type,negate:!f,params:(query:url),type:phrase),query:(match_phrase:(threat.indicator.type:url)))),timeRange:(from:now/d,to:now/d))';
 
-Cypress.Cy.prototype.onUncaughtException = function () {};
+describe('Invalid Indicators', () => {
```

Comment:
this section tests that the table shows up even with missing fields. The following logic is applied:
- test with all fields present
- test with missing `sha256`
- test with missing `sha256` and `md5`
- continue removing fields until there are none left

The UI Indicator name column should display the proper value for the present fields.
If no fields are present, the Indicator name column shows `-`. and the investigate in timeline button is not rendered

Reply:
added more tests for the following:
- missing `threat` field entirely
- missing `type` field
- missing `indicator` field

I think this is overkill but we should be safe